server/welcomebot.go

```diff
@@ -57,7 +57,7 @@ func (p *Plugin) getSiteURL() string {
 	return *config.ServiceSettings.SiteURL
 }
 
-func (p *Plugin) newSampleMessageTemplate(teamName, userID string) (*MessageTemplate, error) {
+func (p *Plugin) newSampleMessageTemplate(teamID string, userID string) (*MessageTemplate, error) {
```

Comment:
The changes in this file come from https://github.com/mattermost/mattermost-plugin-welcomebot/pull/74, right? Would you mind dropping them and we try on https://github.com/mattermost/mattermost-plugin-welcomebot/pull/74 go get merged?

Reply:
@hanzei Yes, doing just that. Will update you once done.

Reply:
These change still seem to come from https://github.com/mattermost/mattermost-plugin-welcomebot/pull/74#discussion_r657143072. Once you dropped them from the file, I'n happy to approve it.